frontend/src/pages/logs/Logs.module.scss

```diff
@@ -55,6 +55,24 @@
       -webkit-filter: blur(1.3px);
     }
 
+    .midContainer::-webkit-scrollbar {
+      width: 10px;
+    }
+    
+    .midContainer::-webkit-scrollbar-track {
+      background-color: $background
+    }
+    
+    .midContainer::-webkit-scrollbar-thumb {
+      box-shadow: inset 0 0 6px $color-mid-grey;
+      border-radius: 5pt;
+      background-color: $dark-blue;
+    }
+    
+    .midContainer::-webkit-scrollbar-corner {
+      background-color: $background;
+    }
+
```

Comment:
I think it would be nice to extract those styles to a seperate shared class (e.g. `customScrollBar`) and reuse that class wherever needed. This would save a lot of copy pasting and would also save a lot of work and bugs if we decide to change some of those scroll bar styles.

An alternative approach would be to implement a `ScrollView` component that would implement those styles. And we could reuse that component in the same way, e.g.:
```tsx
<ScrollView>
{ /* Bellow line is just an example */ }
{arrayOfItems.map(item => <div>...</div>}
</ScrollView>
```

It may be helpfull to always ask yourself: "Does this code need to be reused across other modules/functions/components/classes in the same way?". If the answer is YES, it might be good to find a way to reuse that code. 

The benefits could be:
- less copy pasting (this is the least important one)
- code is easier to change
- enforces a standard implementation approach

Reply:
This article also gives a good overview of when to or not to reuse some piece of code:

https://eledris.com/how-to-write-clean-reusable-code/

Reply:
If you wanna dig even deeper into some of the software engineering practises (not just code reuse), there is this pretty good book ["Clean Code"](https://www.oreilly.com/library/view/clean-code-a/9780136083238/).

Here is a summary of that book: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29